### ISSUE TITLE
Agent: Changelog is missing

### ISSUE THREAD

**Issue by xuhcc** [aragon/aragon-apps#1097]
The "Changelog" link on the app upgrade page ([here](https://mainnet.aragon.org/#/cfdao/apps/installed/agent.aragonpm.eth) for example) leads to https://github.com/aragon/aragon-apps/releases, but there is no changelog. And the latest release was tagged as `apps-agent-v2.1.0` while the app upgrade page states that new version is 5.0.1.

**Comment by sohkai** [comment]
Yes, we should probably update that so it points to a CHANGELOG.md in each app that documents what gets changed between contract updates!
